src/pages/Fox/hooks/useFoxPageContext.tsx

```diff
@@ -0,0 +1,64 @@
+import { type AccountId, type AssetId, fromAssetId } from '@shapeshiftoss/caip'
+import React, { createContext, useContext, useMemo, useState } from 'react'
+import {
+  selectAccountIdByAccountNumberAndChainId,
+  selectAccountNumberByAccountId,
+} from 'state/slices/portfolioSlice/selectors'
+import { useAppSelector } from 'state/store'
+
+type FoxPageContextType = {
+  selectedAssetAccountId: AccountId | undefined
+  assetId: AssetId
```

Comment:
q: This is just passing the passed `assetId` through, do we need/want this?

Reply:
I would like to keep it in place so every consumers are consuming the assetId from a central placew